runcmd.go

```diff
@@ -248,6 +248,9 @@ func (c *runcmd) build(gf *globalFlags, proj *project, b *hatBuilder, r *runner)
 	image := b.baseImage
 	if b.hatPath != "" {
 		image, err = b.applyHat()
+		if err != nil {
+			return err
```

Comment:
can this be moved outside of the other if?

Reply:
I don't think so, this keeps it clear that the error check if for the `b.applyHat()` operation